src/codegen/cfg.rs

```diff
@@ -1201,7 +1203,8 @@ impl ControlFlowGraph {
                         format!("({contract_no}, {function_no})")
                     } else {
                         "_".to_string()
-                    }
+                    },
+                    flags.as_ref().map(|e| self.expr_to_string(contract, ns, e)).unwrap_or("0".to_string())
```

Comment:
I don't think it should be 0 for unspecified, it should be empty (i.e. `.unwrap_or("".to_string())` (maybe `.unwrap_or_default()`.)